EventSubscriber/DeprecatedRouteSubscriber.php

```diff
@@ -49,19 +50,25 @@ public function onKernelResponse(ResponseEvent $event) {
     }
     $response = $event->getResponse();
     // set date in the deprecation response header
-    $deprecationDate = new \DateTime($deprecatedSince);
+    $deprecationDate = \DateTime::createFromFormat(DATE_ATOM, $deprecatedSince);
```

Comment:
Should use DateTimeInterface::ATOM rather than DATE_ATOM.
Also, this check should also be in the annotation class, so that the error is thrown during development, and not during runtime.

Reply:
corrected, but if the check is enforced in the annotation constructor, the exception is thrown on cache:clear as a LoaderLoadException:
HTTP 500 Internal Server Error
Wrong date format, review documentation for the list of accepted formats in /home/marco/projects/bundle-tester2/config/routes/../../src/Controller/ (which is being imported from "/home/marco/projects/bundle-tester2/config/routes/annotations.yaml"). Make sure annotations are installed and enabled.
Is that what you had in mind?